Dockerfile

```diff
@@ -5,6 +5,10 @@ RUN wget -qO- https://deb.nodesource.com/gpgkey/nodesource.gpg.key | apt-key add
 RUN echo 'deb https://deb.nodesource.com/node_0.12 jessie main' > /etc/apt/sources.list.d/nodesource.list
 RUN apt-get update && apt-get install -y nodejs
 
+ENV GEM_HOME=/bundle
+RUN gem update --system
+RUN gem install bundler
```

Comment:
the base image has a different `GEM_HOME`, its not compatible with our tooling. Also I don't think its bundler is current.